src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -572,6 +572,8 @@ private Size UpdateLayout(Size availableSize, bool exactWidth = false)
 						desiredWidth,
 						maxLines: lineAtHeight
 					);
+
+					measuredHeight = Layout.GetLineTop(Layout.LineCount);
```

Comment:
Please add a comment here noting why this is important.

Reply:
Repro is very important indeed, and also please take a look at possible regression in screenshot tests.